eb_sqs/worker/worker_task.py

```diff
@@ -30,15 +30,13 @@ def __init__(self, id, group_id, queue, func, args, kwargs, max_retries, retry,
 
         self.abs_func_name = '{}.{}'.format(self.func.__module__, self.func.__name__)
 
-    def execute(self):
-        # type: () -> Any
+    def execute(self) -> Any:
         from eb_sqs.decorators import func_retry_decorator
         self.func.retry_num = self.retry
         self.func.retry = func_retry_decorator(worker_task=self)
         return self.func(*self.args, **self.kwargs)
 
-    def serialize(self):
-        # type: () -> unicode
+    def serialize(self) -> str:
```

Comment:
Not related to this particular line, but below we have missed the syntax update for the "copy" function